<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WebDownload</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="switchPageFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>520</width>
      <height>40</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <widget class="QPushButton" name="newTaskPushButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>88</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>新建任务</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="taskListPushButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>0</y>
       <width>88</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>任务列表</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="newTaskPageWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>520</width>
      <height>290</height>
     </rect>
    </property>
    <widget class="QPushButton" name="chooseDirPushButton">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>210</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>选择目录</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="dirTextEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>120</y>
       <width>381</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="startPushButton">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>210</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>开始下载</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>下载目录：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="fileTypeComboBox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>210</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="currentText">
      <string>HTML</string>
     </property>
     <item>
      <property name="text">
       <string>HTML</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>文本</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>图像</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe Arabic</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>文件类型：</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="urlTextEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>381</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>网址URL：</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="taskListPageWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>520</width>
      <height>290</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>81</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>任务列表：</string>
     </property>
    </widget>
    <widget class="QListWidget" name="taskListWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>461</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
   </widget>
   <zorder>taskListPageWidget</zorder>
   <zorder>newTaskPageWidget</zorder>
   <zorder>switchPageFrame</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>520</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
